app/services/notification_subscriptions/unsubscribe.rb

```diff
@@ -0,0 +1,42 @@
+module NotificationSubscriptions
+  class Unsubscribe
+    attr_reader :current_user, :permitted_params
+
+    def self.call(...)
+      new(...).call
+    end
+
+    def initialize(current_user, permitted_params)
+      @current_user = current_user
+      @permitted_params = permitted_params
+    end
+
+    def call
+      unsubscribe_subscription
+    end
+
+    def unsubscribe_subscription
+      subscription_id = permitted_params[:subscription_id]
+      unless subscription_id.nil?
+        notification = NotificationSubscription.find_by(user_id: current_user.id,
+                                                        id: subscription_id)
+      end
+      return { errors: "Subscription ID is missing" } if subscription_id.nil?
```

Comment:
Minor: if you move this line up to between 19/20, you don't need the `unless subscription_id.nil?`.